### PR TITLE
Create Controller, Routes, and Views for Neighborhoods

### LINKED ISSUE #16: Create neighborhood controller/basic-views/routes

### PR DESCRIPTION
**Pull request by HacksForHugs**
Finish controller, views, and routes

Fix #16 #26  #38

### CODE REVIEW

**Comment by hidesminimally** [comment]
@HacksForHugs Your tabs should be 2 spaces in Rails code. In your Sublime, go to Views>Indentation and use spaces, and use tab width of 2. Also, you should be using https://github.com/jish/pre-commit!!!! get it A$AP

File: app/controllers/groups_controller.rb [hunk 1]
```diff
@@ -1,13 +1,10 @@
 class GroupsController < ApplicationController
   def index
-  	@groups = Group.all
+    @groups = Group.all
   end
 
   def show
-  	# @group = Group.first
```
Comment (hidesminimally) [inline app/controllers/groups_controller.rb:7]: @HacksForHugs Remove comments if not important in the future!

File: test/controllers/neighborhoods_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (hidesminimally) [inline test/controllers/neighborhoods_controller_test.rb]: @HacksForHugs Remove tests if unnecessary

**Comment by hidesminimally** [comment]
@HacksForHugs added more tags to your PR description. future PR's try to tag all the relevant issues.

File: app/controllers/neighborhoods_controller.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+			if item.title == title
+				return true
+			end
+		end
+		return false	
+	end
+
+	def create
+		@neighborhood = Neighborhood.new
+		@repeat_error = contains_title?(neighhorhood_params[:title])
```
Comment (hidesminimally) [inline app/controllers/neighborhoods_controller.rb]: http://guides.rubyonrails.org/active_record_validations.html#uniqueness

@HacksForHugs This is what you'll need

File: app/controllers/groups_controller.rb (hunk 1 above)
Comment (HacksForHugs) [inline app/controllers/groups_controller.rb:7]: @lilshim I think Nikita created this file, it doesn't exist in my Ashby-village folder. I have no idea how I pushed it up?

**Comment by HacksForHugs** [comment]
@lilshim error message ready for checking

**Comment by hidesminimally** [comment]
LGTM

**Pull request merged**